coffee/lib/abstract-chosen.coffee

```diff
@@ -280,8 +281,17 @@ class AbstractChosen
     if /Android/i.test(window.navigator.userAgent)
       return false if /Mobile/i.test(window.navigator.userAgent)
     return true
+    
+  @tagname_is_valid: : (@form_field) ->
```

Comment:
What's up with this syntax here?  Seems like there's an extra `:` + space.

Additionally, this shouldn't even compile correctly -- did you actually compile the source to test this change in a real Chosen instance?